libs/utils/website/src/lib/auth-link.tsx

```diff
@@ -1,9 +1,14 @@
 import {ApolloLink, DefaultContext} from '@apollo/client'
 import {AuthTokenStorageKey} from '@wepublish/authentication/website'
 import {getCookie} from 'cookies-next'
+import {Observable} from 'zen-observable-ts'
 
 export const authLink = new ApolloLink((operation, forward) => {
-  const {token} = JSON.parse(getCookie(AuthTokenStorageKey)?.toString() ?? '{}')
+  const {token: cookieToken} = JSON.parse(getCookie(AuthTokenStorageKey)?.toString() ?? '{}')
+  const {token: sessionStorageToken} = JSON.parse(
```

Comment:
This was copied over from hauptstadt, hauptstadt has the token in session storage as the cookie is httpOnly. Left it here as it makes no difference.